src/main/twirl/uk/gov/hmrc/play/views/formatting/Dates.scala

```diff
@@ -38,11 +42,27 @@ object Dates {
 
   def formatDateTime(date: DateTime) = dateFormat.print(date)
 
-  def formatEasyReadingTimestamp(date: Option[DateTime], default: String) = date match {
-    case Some(d) => {
-      s"${easyReadingTimestampFormat.print(d).toLowerCase}, ${easyReadingDateFormat.print(d)}"
+  def formatEasyReadingTimestamp(date: Option[DateTime], default: String)(implicit lang: Lang) = {
+
+    def englishEasyDate = date match {
```

Comment:
```scala
val englishEasyDate: DateTime => String = d => s"${easyReadingTimestampFormat.print(d).toLowerCase}, ${easyReadingDateFormat.print(d)}"
val welshEasyDate: DateTime => String = d => s"${easyReadingTimestampFormat.print(d).toLowerCase}, ${WeekDaysInWelsh(d.getDayOfWeek)} ${d.getDayOfMonth} ${MonthNamesInWelsh(d.getMonthOfYear)} ${d.getYear}"
val formatter = lang.code match {
 case "cy" => welshEasyDate
 case _ => englishEasyDate
}
date.fold(default)(formatter)
```

this defines two function values for formatting the date english/welsh
then picks one of the functions as the "formatter", lastly, applies a `default` for when date is not specified, or formats the date using appropriate formatter